### ISSUE TITLE
Cannot build the project on Intel Mac and IntelliJ

### ISSUE THREAD

**Issue by PattaFeuFeu** [andregasser/bigbone#269]
After importing the project to work on one of the issues, I already ran into build problems, unfortunately. 😕 

```
> Task :buildSrc:jar UP-TO-DATE

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':bigbone'.
> Failed to calculate the value of task ':bigbone:compileJava' property 'javaCompiler'.
   > No matching toolchains found for requested specification: {languageVersion=8, vendor=any, implementation=vendor-specific} for MAC_OS on x86_64.
      > No locally installed toolchains match and toolchain download repositories have not been configured.
```

**Do you have any tips on what I should have installed or how my setup should look like?**

I’m also willing to upgrade the README to include those tips. I just need to know where to start. 😊 

```
IntelliJ IDEA 2023.2.2 (Community Edition)
Build #IC-232.9921.47, built on September 12, 2023
Runtime version: 17.0.8+7-b1000.22 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 12
Metal Rendering is ON
Registry:
    ide.experimental.ui=true


Kotlin: 232-1.9.0-IJ9921.47
```

**Comment by PattaFeuFeu** [comment]
This may need Gradle’s [Toolchains for JVM projects](https://docs.gradle.org/8.0.2/userguide/toolchains.html#sub:download_repositories) support set up properly to work on different OSes and with different installations 🤔

**Comment by andregasser** [comment]
Hello @PattaFeuFeu 
Thanks for the feedback and the fix :-)

I have created a new PR for improving the Gradle build a bit (see https://github.com/andregasser/bigbone/pull/274). Based on your feedback, I have also added the foojay resolver to `settings.gradle`. 

I do not have a macOS here where I am, so I have to wait until end of this week. I would like to test the build on my Mac bevore I merge this into master. But feel free to test out the changes in this PR on your machine. Happy to get some feedback, whether  the build script changes work for your case or not.

Best regards,
André
